ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/forkchoice/ForkChoiceBlobSidecarsAvailabilityChecker.java

```diff
@@ -106,34 +110,34 @@ public SafeFuture<BlobSidecarsAndValidationResult> getAvailabilityCheckResult()
   @Override
   public BlobSidecarsAndValidationResult validateImmediately(final List<BlobSidecar> blobSidecars) {
 
-    final List<KZGCommitment> kzgCommitments = kzgCommitmentsSupplier.get();
+    final List<KZGCommitment> kzgCommitmentsFromBlock = kzgCommitmentsFromBlockSupplier.get();
 
     if (!blobSidecars.isEmpty()) {
-      return validateBatch(blobSidecars, kzgCommitments);
+      return validateBatch(blobSidecars);
```

Comment:
We are not doing completeness check in this case or I missed it.

Reply:
yes, good catch, I wasn't clearly in a good shape!